coralproject/talk
# Reject Rate Fixes

## What does this PR do?

- Corrected query to select all statuses (default was accepted/none)
- Fixed up some missing prop-types
- Improved count code

## How do I test this PR?

Visit the admin panel, check on a user that has comments, and click their username to see the reject rate corrected.
